### DISCUSSION TITLE
LTS Version of primeNG with invalid license

### DISCUSSION THREAD

**Discussion by amoltech88** [primefaces/community#1502] [PrimeNG] [answered]
I've acquired the LTS license for PrimeNG and successfully activated it. Additionally, I've assigned the license to version 14 to align with my usage of Angular 14.2.2. Despite integrating the license key and passcode using LicenseManager.verify('key','passcode') in main.ts, the application still indicates an invalid license for the PrimeNG LTS version.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Could you please send an e-mail to contact@primetek.com.tr ?
